lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -186,10 +198,13 @@ def refund(money, intent_id, options = {})
       def store(payment_method, options = {})
         params = {}
         post = {}
-
         # If customer option is provided, create a payment method and attach to customer id
         # Otherwise, create a customer, then attach
-        if payment_method.is_a?(StripePaymentToken) || payment_method.is_a?(ActiveMerchant::Billing::CreditCard)
+        if payment_method.is_a?(ActiveMerchant::Billing::NetworkTokenizationCreditCard)
+          tokenize_apple_google = tokenize_apple_google(payment_method, options)
```

Comment:
🛑  Warning:

naming a variable with the same name that a method can cause some issues with code execution or IDE tooling.